Backtesting/collectors/exchanges/exchange_history_collector/history_collector.py

```diff
@@ -38,34 +39,45 @@ def __init__(self, config, exchange_name, symbols, time_frames, use_all_availabl
         self.exchange_manager = None
 
     async def start(self):
-        self.exchange_manager = await trading_api.create_exchange_builder(self.config, self.exchange_name) \
-            .is_simulated() \
-            .is_rest_only() \
-            .is_exchange_only() \
-            .is_collecting() \
-            .is_ignoring_config() \
-            .disable_trading_mode() \
-            .build()
-
-        self.exchange = self.exchange_manager.exchange
-        self._load_timeframes_if_necessary()
-
-        # create description
-        await self._create_description()
-
-        self.logger.info("Start collecting history")
-        for symbol in self.symbols:
-            self.logger.info(f"Collecting history for {symbol}...")
-            await self.get_ticker_history(self.exchange_name, symbol)
-            await self.get_order_book_history(self.exchange_name, symbol)
-            await self.get_recent_trades_history(self.exchange_name, symbol)
-
-            for time_frame in self.time_frames:
-                self.logger.info(f"Collecting history on {time_frame}...")
-                await self.get_ohlcv_history(self.exchange_name, symbol, time_frame)
-                await self.get_kline_history(self.exchange_name, symbol, time_frame)
-
-        await self.stop()
+        stop_database = True
```

Comment:
I would have prefer : 
```suggestion
        should_stop_database = True
```